src/Raygun4php/RaygunClient.php

```diff
@@ -304,6 +342,61 @@ private function is_assoc($array)
       return (bool)count(array_filter(array_keys($array), 'is_string'));
     }
 
+    /**
+     * flushBundle
+     *
+     * Flush any leftover messages stored in the global session which weren't bundled
+     *
+     * @return [void]
+     */
+
+    private function flushBundle() {
+      $bundle = $this->bundler->getFromStorage();
+
+      if(!empty($bundle)) {
+        $this->bundler->setBundle($bundle);
+
+        if(count($bundle) === 1) {
+          $this->Send($bundle[0]);
+        }
+        else {
+          $this->SendBundle($bundle);
```

Comment:
`SendBundle` doesn't accept any parameters. Also if we are retrieving it from storage couldn't we just add it to the current bundle and wait for it queued up by subsequent errors? 

Perhaps this was intended to send the errors immediately after retrieving them again.